src/game_state/player.rs

```diff
@@ -77,6 +78,52 @@ pub fn start_player_state(
                 let mut player = msg.player;
                 player.entity_id = players.len().try_into().expect("too many players");
                 println!("{:?}", player);
+                let join_game = JoinGame {
+                    entity_id: player.entity_id,
+                    gamemode: 1,
+                    dimension: 0,
+                    difficulty: 0,
+                    max_players: 2,
+                    level_type: String::from("default"),
+                    reduced_debug_info: false,
+                };
+                send_packet!(messenger, msg.conn_id, Packet::JoinGame(join_game)).unwrap();
+                let player_pos_and_look = ClientboundPlayerPositionAndLook {
+                    x: player.position.x,
+                    y: player.position.y,
+                    z: player.position.z,
+                    yaw: 0.0,
+                    pitch: 0.0,
+                    flags: 0,
+                    teleport_id: 0,
+                };
+                send_packet!(
+                    messenger,
+                    msg.conn_id,
+                    Packet::ClientboundPlayerPositionAndLook(player_pos_and_look)
+                )
+                .unwrap();
+                send_packet!(
+                    messenger,
+                    msg.conn_id,
+                    Packet::ChunkData(ChunkData {
+                        chunk_x: 0,
+                        chunk_z: 0,
+                        full_chunk: true,
+                        primary_bit_mask: 1,
+                        size: 12291, //I just calculated the length of this hardcoded chunk section
+                        data: ChunkSection {
+                            bits_per_block: 14,
+                            data_array_length: 896,
+                            block_ids: Vec::new(),
+                            block_light: Vec::new(),
+                            sky_light: Vec::new(),
+                        },
+                        biomes: vec![127; 256],
+                        number_of_block_entities: 0,
+                    })
```

Comment:
This needs to be replaced with an actual call to the state reporter. There's an order issue in the login protocol we need to fix. There's a ticket in the backlog describing it

Reply:
https://github.com/DuncanUszkay1/Patchwork/issues/47